subprojects/docs/src/snippets/buildCache/integration-tests/groovy/build.gradle

```diff
@@ -2,7 +2,14 @@ plugins {
     id('java-library')
 }
 
-tasks.register("integTest", Test)
+sourceSets {
+    integTest
+}
+
+tasks.register("integTest", Test) {
+    classpath = sourceSets.integTest.runtimeClasspath
+    testClassesDirs = sourceSets.integTest.output.classesDirs
+}
 
```

Comment:
Maybe just create this via test suites?

Reply:
I was under the impression that we should not use incubating APIs in samples/snippets.